0.0.103 CHANGELOG tweaks from Jeff

TheBlueMatt opened the pull request:
Followup on https://github.com/rust-bitcoin/rust-lightning/pull/1153#pullrequestreview-796053056 though still need to address a few things from there pending response from Jeff.

jkczyz commented on CHANGELOG.md:
```diff
@@ -36,11 +37,12 @@
-   messages between pings to ensure we should always receive pongs in a timely
-   manner. Together, these should significantly reduce instances of us failing
-   to remain connected to a peer during initial gossip sync (#1137).
+ * Previous versions would often disconnect peers during initial graph sync due
+   to ping timeouts while processing large numbers of gossip messages. We now
+   delay disconnecting peers if we receive messages from them even if it takes
+   a while to receive a pong from them. Further, we avoid sending too many
+   gossip messages between pings to ensure we should always receive pongs in a
+   timely manner (#1137).
  * If a payment is sent, creating an outbound HTLC and sending it to our
    counterparty (implying the `ChannelMonitor` was persisted on disk), but the
    `ChannelManager` was not persisted prior to shutdown/crash, no
```
My feedback was more about the second bullet just describing the bug and not the fix. Maybe that's ok but the use of the word "will" instead of "would" makes it seem like it's describing the fix.

jkczyz commented on CHANGELOG.md:
```diff
@@ -46,8 +46,9 @@
    `ChannelManager` was not persisted prior to shutdown/crash, no
-   `Event::PaymentPathFailed` event will be generated if the HTLC is eventually
-   failed on chain (#1104).
+   `Event::PaymentPathFailed` event was be generated if the HTLC is eventually
```
s/was be/was
s/is/

TheBlueMatt replied:
I tweaked the case to be past-tense more consistently.

jkczyz commented:
Oh, and `PaymentId` was added in #1059. Just noticed your comment on #1153.

jkczyz commented on CHANGELOG.md:
```diff
@@ -43,10 +43,10 @@
    counterparty (implying the `ChannelMonitor` was persisted on disk), but the
    `ChannelManager` was not persisted prior to shutdown/crash, no
-   `Event::PaymentPathFailed` event was be generated if the HTLC is eventually
+   `Event::PaymentPathFailed` event was generated if the HTLC is eventually
```
Also, "is" here could be dropped or changed to "was"

TheBlueMatt replied:
I think I beat you to this one :)

jkczyz replied:
lol, my original comment was actually about the "is" in "HTLC is"... I missed the first "is". 😛

TheBlueMatt commented:
Squashed without diff, updated the release notes on github with the changes but will land this after CI.

```
$ git diff-tree -U1 ba4fc7f14 7df087a36
$
```
